File: LocalShare/Models/SignallingEvents.cs
namespace LocalShare.Models
{
    public static class SignallingEvents
    {
        public const string UpdateSelf = "UpdateSelf";
        public const string AddConnectedClient = "AddConnectedClient";
        public const string ReceiveOffer = "ReceiveOffer";
        public const string ReceiveAnswer = "ReceiveAnswer";
        public const string ReceiveIceCandidate = "ReceiveIceCandidate";
        public const string RemoveDisconnectedClient = "RemoveDisconnectedClient";
    }
}

File: LocalShare/ClientApp/src/models/SignallingEvents.ts
export enum SignallingEvents {
  UpdateSelf = "UpdateSelf",
  AddConnectedClient = "AddConnectedClient",
  ReceiveOffer = "ReceiveOffer",
  ReceiveAnswer = "ReceiveAnswer",
  ReceiveIceCandidate = "ReceiveIceCandidate",
  RemoveDisconnectedClient = "RemoveDisconnectedClient",
  SendIceCandidate = "SendIceCandidate",
  SendAnswer = "SendAnswer",
  SendOffer = "SendOffer",
}
